setup.py

```diff
@@ -62,7 +59,7 @@
 install_requires = [
     'Flask>=0.11.1',
     'Flask-BabelEx>=0.9.2',
```

Comment:
This needs to be bumped to `0.9.3` because of [`invenio-accounts`](https://github.com/inveniosoftware/invenio-accounts/blob/e86c8994eedb62a9769241ec2e7e118d818f49b9/setup.py#L69)